app/src/modules/fota/Kconfig.fota

```diff
@@ -11,15 +11,15 @@ menuconfig APP_FOTA
 
 config APP_FOTA_THREAD_STACK_SIZE
 	int "Thread stack size"
-	default 2048
```

Comment:
Will you revert:
https://github.com/hello-nrfcloud/firmware/commit/324fd01be0c7634a2479714a3cc8839d65f7e332

now that FMFU is non-blocking?

Reply:
Will do